### ISSUE TITLE
HTML elements with insufficient color contrast in The Agency Theme

### ISSUE THREAD

**Issue by DemeSzabolcs** [OrchardCMS/OrchardCore#11241]
### Describe the bug
There are HTML elements with insufficient colors in [The Agency Theme](https://github.com/OrchardCMS/OrchardCore/tree/main/src/OrchardCore.Themes/TheAgencyTheme). These are accessibility errors.

 These may not be all of the errors, but most of them:

```
Asserting the accessibility analysis result failed. Check the accessibility report failure dump for details.
-------- Shouldly.ShouldAssertException : axeResult.Violations
    should be empty but had
1
    item and was
[Selenium.Axe.AxeResultItem (33056876)]

Additional Info:
    Elements must have sufficient color contrast: 
    <a class="btn btn-primary btn-xl text-uppercase" href="#services">Tell Me More</a>
    <h3 class="section-subheading text-muted">Lorem ipsum dolor sit amet consectetur.</h3>
    <h3 class="section-subheading text-muted">Lorem ipsum dolor sit amet consectetur.</h3>
    <p class="text-muted">Lead Designer</p>
    <p class="text-muted">Lead Marketer</p>
    <p class="text-muted">Lead Developer</p>
    <p class="large text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut eaque, laboriosam veritatis, quos non quis ad perspiciatis, totam corporis ea, alias ut unde.</p>
    <button class="btn btn-primary btn-xl text-uppercase" id="sendMessageButton" type="submit">Send Message</button>
    <a class="navbar-brand" href="/#page-top">Test Site</a>
    <a class="nav-link" href="/#services">Services</a>
    <a class="nav-link" href="/#portfolio">Portfolio</a>
    <a class="nav-link" href="/#about">About</a>
    <a class="nav-link" href="/#team">Team</a>
    <a class="nav-link" href="/#contact">Contact</a>
    <div class="masthead-subheading">Welcome To Our Studio!</div>
    <div class="masthead-heading text-uppercase">Test Site</div>
    <h4>2009-2011</h4>
    <h4 class="subheading">Our Humble Beginnings</h4>
    <p class="text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt ut voluptatum eius sapiente, totam reiciendis temporibus qui quibusdam, recusandae sit vero unde, sed, incidunt et ea quo dolore laudantium consectetur!</p>
    <h4>March 2011</h4>
    <h4 class="subheading">An Agency is Born</h4>
    <p class="text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt ut voluptatum eius sapiente, totam reiciendis temporibus qui quibusdam, recusandae sit vero unde, sed, incidunt et ea quo dolore laudantium consectetur!
</p>
    <h4>December 2015</h4>
    <h4 class="subheading">Transition to Full Service</h4>
    <p class="text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt ut voluptatum eius sapiente, totam reiciendis temporibus qui quibusdam, recusandae sit vero unde, sed, incidunt et ea quo dolore laudantium consectetur!</p>
    <h4>July 2020</h4>
    <h4 class="subheading">Phase Two Expansion</h4>
    <p class="text-muted">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Sunt ut voluptatum eius sapiente, totam reiciendis temporibus qui quibusdam, recusandae sit vero unde, sed, incidunt et ea quo dolore laudantium consectetur!
</p>
    <h4>
                        Be Part
                        <br>Of Our
                        <br>Story!
                    </h4>
    <h2 class="section-heading text-uppercase">Contact Us</h2>
    <h3 class="section-subheading text-light">Lorem ipsum dolor sit amet consectetur.</h3>
```

### To Reproduce
1. Set up Orchard Core with the [agency recipe](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Themes/TheAgencyTheme/Recipes/agency.recipe.json).
2. Login and go to the homepage.
3. See that there are accessibility problems on the site, mainly insufficient color contrast.

### Expected behavior
After setting up Orchard Core with the [agency recipe](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Themes/TheAgencyTheme/Recipes/agency.recipe.json) there should be no accessibility problems on the homepage.

**Comment by Skrypt** [comment]
We are not the creators of these themes. Suggestion needs to be made one this repository : https://github.com/StartBootstrap/startbootstrap-agency

**Comment by hishamco** [comment]
I'm agree with @Skrypt, we used the theme as it is
